packages/dcos-image-deps/build

```diff
@@ -4,9 +4,9 @@ export LIB_INSTALL_DIR="$PKG_PATH/lib/python3.5/site-packages"
 mkdir -p "$LIB_INSTALL_DIR"
 
 # Install wheels.
-for package in analytics-python azure-nspkg azure-common azure-mgmt-nspkg \
-    azure-mgmt-network azure-storage beautifulsoup4 certifi docutils enum34 keyring msrest \
-    msrestazure py requests-oauthlib schema webob; do
+for package in adal analytics-python azure-nspkg azure-common azure-mgmt-nspkg \
```

Comment:
adal? Is it related to the pytest bump?

Reply:
Yes, all new packages here were shown as required by the CI.